stockflux-securities-master/src/components/InputForm.css

```diff
@@ -104,12 +104,20 @@
   width: 280px;
 }
 
-.input-submit-button > button:focus {
+.input-submit-button.in-progress > button {
   outline: none;
   color: var(--text-grey-darkest);
   border-color: var(--text-grey-darkest);
 }
 
+.input-submit-button > button:focus {
+  outline: none;
+}
+
+
```

Comment:
Can we remove this excess space, please?

Reply:
done